docs/en/latest/plugins/opentelemetry.md

```diff
@@ -28,31 +29,59 @@ description: This document contains information about the Apache opentelemetry P
-| sampler.options.root.name             | string        | False    | always_off                                      | ["always_on", "always_off", "trace_id_ratio"]                | Root sampling strategy.                                                                                                                                                                                                                      |
-| sampler.options.root.options          | object        | False    | {fraction = 0}                                  |                                                              | Root sampling strategy parameters.                                                                                                                                                                                                           |
-| sampler.options.root.options.fraction | number        | False    | 0                                               | [0, 1]                                                       | Root sampling probability for `trace_id_ratio`.                                                                                                                                                                                              |
-| additional_attributes                 | array[string] | False    |                                                 |                                                              | Variables and its values which will be appended to the trace span.                                                                                                                                                                           |
-| additional_attributes[0]              | string        | True     |                                                 |                                                              | APISIX or Nginx variables. For example, `http_header` or `route_id`.                                                                                                                                                                         |
+The plugin only supports binary-encoded [OLTP over HTTP](https://opentelemetry.io/docs/reference/specification/protocol/otlp/#otlphttp) currently.
+
+:::
+
+## Load Plugin
+
+APISIX doesn't load the `opentelemetry` plugin by default because of the `conf/config-default.yaml` file. Follow the steps to load it:
 
-### Configuring the collector
+1. Open the `conf/config-default.yaml` file, and copy the `plugins` field with **all plugin names** to the `conf/config.yaml` file.
+2. Uncomment the `#- opentelemetry` line or remove the `#`.
+3. Follow the [Plugin Hot Reload Guide](/docs/apisix/terminology/plugin/#hot-reload) to reload plugins.
 
-You can set up the collector by configuring it in you configuration file (`conf/config.yaml`):
+```yaml title="conf/config.yaml"
+plugins:
+  - ...
+  - opentelemetry
+  - ...
+```
+
+## Plugin YAML Attributes
```

Comment:
I was using `Plugin Meta Attributes` to refer to configurations under the `conf/config.yaml` file, but I come up with the `_meta` attributes in plugin, so I use `Plugin YAML Attributes` instead, but I think it's still wired 🤔 

Do you have a better choice? cc @navendu-pottekkat @nfrankel @tzssangglass @bzp2010

Reply:
`Plugin Metadata` is ok for me